### ISSUE TITLE
ARM64: Assert failure: No method entry found for System.StubHelpers.StubHelpers::MulticastDebuggerTraceHelper. FAILED: 0 != id

### ISSUE THREAD

**Issue by kyulee1** [dotnet/coreclr#6568]
This happens crossgen on system.private.corlib build.

```
CROSSGEN! CHECK::Trigger + 0x254 (0x00007ff6`ebf9f3f4)
CROSSGEN! FindECFuncForMethod + 0xD4 (0x00007ff6`ebd5e754)
CROSSGEN! ECall::GetFCallImpl + 0x1CF (0x00007ff6`ebd5ecef)
CROSSGEN! MethodDesc::Save + 0x7A7 (0x00007ff6`ebc22b87)
CROSSGEN! MethodDesc::SaveChunk::SaveOneChunk + 0x28C (0x00007ff6`ebc2325c)
CROSSGEN! MethodDesc::SaveChunk::Save + 0x2BC (0x00007ff6`ebc22fac)
CROSSGEN! EEClass::Save + 0x326 (0x00007ff6`ebc78d96)
CROSSGEN! MethodTable::Save + 0xBDD (0x00007ff6`ebc710ad)
CROSSGEN! Module::SaveMethodTable + 0x46 (0x00007ff6`ebd0ced6)
CROSSGEN! Module::Save + 0xBB8 (0x00007ff6`ebd0b6e8)
    File: e:\githubrepo\coreclr\src\vm\ecall.cpp Line: 270
    Image: E:\GithubRepo\coreclr\bin\Product\Windows_NT.arm64.Debug\x64\crossgen.exe
```

**Comment by gkhanna79** [comment]
This indicates that Spc.dll and coreclr.dll are out of sync. Are you able to repro this locally? And if you revert change from Priya, does it repro?

**Comment by kyulee1** [comment]
Yes. I can repro it. It's from Priya's change. I understand the issue. Cross-component of clrjit.dll is incorrectly built -- x64 to x64 instead of x64 to arm64. I'm testing the fix in #6564.

**Comment by kyulee1** [comment]
Fixed via #6564
